.gitignore

```diff
@@ -1 +1 @@
-/Build
+/Build
```

Comment:
nit: not a big deal at all, but the red icon is here because there is no newline at the end